weasyl/search.py

```diff
@@ -31,6 +31,8 @@
     "journal": (30, "j", "journal", "journalid", 3999),
 }
 
+count_limit = 10000
```

Comment:
Not sure. The capitalization is intentional (I’m less and less sure of the value of formatting constants like that when most variables could be immutable anyway), but I don’t know if it’s the right choice.